src/logic/wallets/getWeb3.ts

```diff
@@ -27,33 +26,6 @@ export const WALLET_PROVIDER = {
   TREZOR: 'TREZOR',
 }
 
-export enum ExplorerTypes {
```

Comment:
I would suggest doing a mix between the current implementation 
```
export const getExplorerInfo = (hash: string): BlockScanInfo => {
  const { name, url } = getNetworkExplorerInfo()

  const blockScanInfo = () => {
    const type = hash.length > 42 ? 'tx' : 'address'

    return  {
      url: `${url}${type}/${hash}`,
      alt:  name || '',
    }
  }

  return blockScanInfo
}
```
and deleted one.

I think is a good idea to define `ExplorerTypes` per network explorer. (etherscan uses `/tx` but blockscout could use  `/transaction` instead)

Then we could define strategies per each type explorer (belonging to a network) and regarding the current network returning the link to the explorer

Reply:
Sorry @nicosampler, I followed the discussion where you started it, on private. Let's continue there and summarize here if needed.

Reply:
Created #1450 to do this